### DISCUSSION TITLE
Adapter recommendations?

### DISCUSSION THREAD

**Discussion by ErnieBall** [pvvx/THB2#98] [General]
Im looking for recommendations on which USB-COM adapter to buy. 
Im not really sure what to look for, so im worried i buy the wrong thing.

If  you could share one or more adapter models that ar proven to work for flashing these devices, that would be very helpful.

Thanks!

**Comment by pvvx** [comment]
I have many different USB-COM adapters.
I usually use the one that is closest or the first one I found in the box where they are stored.
The last one I used was this one:
![image](https://github.com/user-attachments/assets/557ed0bb-f028-49a4-9073-4ea665e62510)
An additional connector is soldered to the RTS output.

**Reply by ErnieBall** [reply]
Thanks man, but I was hoping to get a model number or something I can search for on Ali .

**Reply by pvvx** [reply]
This photo is taken from AliExpress for identification.
